app/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "NODE_OPTIONS='--inspect' ts-node --project tsconfig.server.json server.ts",
+    "dev": "nodemon",
```

Comment:
is it intentional? I recall we discussed using `nodemon` ...

Reply:
Yes we snuck nodemon into this ticket since it should have been set up at the beginning of the project and I needed to restart my server a lot for this PR. This dev command got moved to `nodemon.json`.